wiv.js

```diff
@@ -244,14 +245,14 @@ function wiv(params) {
     }
 
     // draw right
-    for (; y <= bottomRight.y; y += increment) {
+    for (; y <= bottomRight.y - thicknessPadding; y += increment) {
       x = calculateRightXValue(y);
       imageMode &&  Math.floor(y % imageFrequency) == 0 && ctx.drawImage(canvasImage, x , y , imageSize, imageSize);
       ctx.lineTo(x, y);
     }
 
     // draw bottom
-    for (; x >= bottomLeft.x; x -= increment) {
+    for (; x >= bottomLeft.x + thicknessPadding; x -= increment) {
```

Comment:
Should we do this for the top corners too, even if the issue isn't as apparent there? Or is there a reason this is only necessary for the bottom corners?